Projects/App/Sources/UI/Place/PlaceList/View/Views/SegmentHeaderView.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  SegmentHeaderView.swift
+//  App
+//
+//  Created by 리아 on 2022/11/01.
+//  Copyright © 2022 com.zesty. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+final class SegmentHeaderView: UICollectionReusableView {
+    
+    // MARK: - Properties
+    weak var viewModel: PlaceListViewModel?
+    weak var addPlaceDelegate: AddPlaceDelegate?
+    
+    private let segmentIndicator = UIView()
+    private let segmentedControl = UISegmentedControl(items: ["전체", "선정맛집"])
+    
+    private let questionButton = UIButton()
+    private let addPlaceButton = UIButton()
+    
+    // MARK: - Life Cycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configureUI()
+        createLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+}
+
+// MARK: - UI Functions
+
+extension SegmentHeaderView {
+    
+    @objc func addPlaceButtonTapped() {
+        addPlaceDelegate?.addPlaceButtonTapped()
+    }
+    
+    @objc func indexChanged(_ sender: UISegmentedControl) {
+        let numberOfSegments = CGFloat(segmentedControl.numberOfSegments)
+        let selectedIndex = CGFloat(sender.selectedSegmentIndex)
+        let titlecount = CGFloat((segmentedControl.titleForSegment(at: sender.selectedSegmentIndex)!.count))
+        
+        if selectedIndex == 1 {
+            viewModel?.placeType = .hot
+            segmentIndicator.snp.remakeConstraints { (make) in
+                make.top.equalTo(segmentedControl.snp.bottom).offset(3)
+                make.height.equalTo(3)
+                make.width.equalTo(titlecount * 11)
+                make.centerX.equalTo(segmentedControl.snp.centerX).dividedBy(numberOfSegments / CGFloat(2.58 + CGFloat(selectedIndex-1.0)*2.0))
+            }
+        } else {
+            viewModel?.placeType = .whole
+            segmentIndicator.snp.remakeConstraints { make in
+                make.top.equalTo(segmentedControl.snp.bottom).offset(3)
+                make.height.equalTo(3)
+                make.width.equalTo(20)
+                make.centerX.equalTo(segmentedControl.snp.centerX).dividedBy(segmentedControl.numberOfSegments)
+            }
+        }
+
+        UIView.animate(withDuration: 0.3) {
+            self.layoutIfNeeded()
+            self.segmentIndicator.transform = CGAffineTransform(scaleX: 1.1, y: 1)
+        } completion: { _ in
+            UIView.animate(withDuration: 0.4, animations: {
+                self.segmentIndicator.transform = CGAffineTransform.identity
+            })
+        }
+    }
+
+}
+
+extension SegmentHeaderView {
+    
+    private func configureUI() {
+        backgroundColor = .white
+        
+        removeBackgroundAndDivider()
+        segmentIndicator.backgroundColor = .black
```

Comment:
p1;

여기 저번에 닉 PR에서 빼먹은 부분인데 라운드 추가 부탁드립니다🙇‍♂️

```suggestion
        segmentIndicator.backgroundColor = .black
        segmentIndicator.layer.cornerRadius = 1
        segmentIndicator.layer.masksToBounds = true
```

Reply:
6de1644156365e42a2085a256102ad0df8a01944 추가했습니다 ✨